### ISSUE TITLE
Patch failed with recent commits.

### ISSUE THREAD

**Issue by smallzhan** [d12frosted/homebrew-emacs-plus#146]
`brew install emacs-plus --HEAD --with-xwidgets --with-jansson --without-spacemacs-icon`
with the commit id `bcc3d36d8a`.

The patch `xwidgets_webkit_in_cocoa.patch` failed on file `src/xwidget.c` 

>patching file src/xwidget.c
Hunk \# 1 FAILED at 18.
Hunk \# 2 succeeded at 67 (offset 1 line).
Hunk \# 3 succeeded at 81 (offset 1 line).
Hunk \# 4 succeeded at 99 (offset 1 line).
Hunk \# 5 succeeded at 117 (offset 1 line).
Hunk \# 6 succeeded at 176 (offset 1 line).
Hunk \# 7 succeeded at 214 (offset 1 line).
Hunk \# 8 succeeded at 253 (offset 1 line).
Hunk \# 9 succeeded at 269 (offset 1 line).
Hunk \# 10 succeeded at 305 (offset 1 line).
Hunk \# 11 succeeded at 563 (offset 1 line).
Hunk \# 12 succeeded at 573 (offset 1 line).
Hunk \# 13 succeeded at 587 (offset 1 line).
Hunk \# 14 succeeded at 645 (offset 1 line).
Hunk \# 15 succeeded at 661 (offset 1 line).
Hunk \# 16 succeeded at 736 (offset 1 line).
Hunk \# 17 succeeded at 754 (offset 1 line).
Hunk \# 18 FAILED at 777.
Hunk \# 19 succeeded at 846 (offset 2 lines).
Hunk \# 20 succeeded at 885 (offset 2 lines).
Hunk \# 21 succeeded at 929 (offset 2 lines).
Hunk \# 22 succeeded at 940 (offset 2 lines).
Hunk \# 23 succeeded at 951 (offset 2 lines).
Hunk \# 24 succeeded at 972 (offset 2 lines).
Hunk \# 25 succeeded at 981 (offset 2 lines).
Hunk \# 26 succeeded at 991 (offset 2 lines).
Hunk \# 27 succeeded at 1017 (offset 2 lines).
Hunk \# 28 succeeded at 1100 (offset 2 lines).
Hunk \# 29 succeeded at 1218 (offset 2 lines).
Hunk \# 30 succeeded at 1392 (offset 2 lines).
Hunk \# 31 succeeded at 1421 (offset 2 lines).
Hunk \# 32 succeeded at 1454 (offset 2 lines).
Hunk \# 33 succeeded at 1468 (offset 2 lines).
2 out of 33 hunks FAILED -- saving rejects to file src/xwidget.c.rej
patching file src/xwidget.h
Error: Failure while executing; `patch -g 0 -f -p1 -i /private/tmp/emacs-plus--patch-20190714-69601-1s5ipq/xwidgets_webkit_in_cocoa.patch` exited with 1.

**Comment by fuxialexander** [comment]
You can try this new patch: https://gist.github.com/fuxialexander/9c69f0d6306e0f8247255999e2db7754

sha: 53c32da5f577a9f0617207c4ef9828498ef746d5eb18200de578fc6d0746d6e2

simply use `brew edit emacs-plus` to edit the formula and replace the corresponding URL and sha

**Comment by d12frosted** [comment]
@fuxialexander you can send this patch as PR :) 

https://github.com/d12frosted/homebrew-emacs-plus/blob/master/patches/xwidgets_webkit_in_cocoa.patch

**Comment by kirill-gerasimenko-zocdoc** [comment]
Patch applies fine, but build is failing for me

```
make[1]: *** No rule to make target `nsxwidget.m', needed by `gl-stamp'.  Stop.
make: *** [src] Error 2
make: *** Waiting for unfinished jobs....
```

**Comment by fuxialexander** [comment]
@kirill-gerasimenko-zocdoc Sorry, the patch missed two newly created file. Please have a try again.

**Comment by pcr910303** [comment]
@d12frosted Actually, I have my patches pending in the emacs-dev mailing list: Please see https://lists.gnu.org/archive/html/emacs-devel/2019-07/msg00640.html
Hopefully the patch gets merged and we wont need any patches.
Until them, using them (instead of the giant patch I have added before) might be better. I made some fixes to the patch that isn’t in the current patches.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by hanfeng0409** [comment]
> You can try this new patch: https://gist.github.com/fuxialexander/9c69f0d6306e0f8247255999e2db7754
> 
> sha: ca85e3b44b5ce88350318f9f16aaee4a91c4d8fb97fc111d5c7f4e9e6bbfe466
> 
> simply use `brew edit emacs-plus` to edit the formula and replace the corresponding URL and sha

@fuxialexander I replace the url  with "https://gist.githubusercontent.com/fuxialexander/9c69f0d6306e0f8247255999e2db7754/raw/68cc2d799e4e4996bf5dccdce3c8f949c4f007c5/new_xwidget_patch.patch" and and she, it tell me that she mismatch. What I should I do?

**Comment by d12frosted** [comment]
Was fixed by #166
